bitbots_body_behavior/src/bitbots_body_behavior/main.dsd

```diff
@@ -1,6 +1,9 @@
 #SearchBall
 @ChangeAction + action:searching, @LookAtFieldFeatures, @StandAndWait + duration:4, @TurnAround
 
+#SearchBallGoalie
```

Comment:
I think the goalie should go to its `@GoToRolePosition` instead of facing the enemy goal at the current location. Going to the center of our goal and face the enemy side is a safe spot, if we don't know what to do. Facing the enemy goal at a position outside of our goal introduces some blind areas. It also means less code.